Source/Scene/PointCloud3DTileContent.js

```diff
@@ -898,6 +913,7 @@ define([
 
         var vs = 'attribute vec3 a_position; \n' +
                  'varying vec4 v_color; \n' +
+                 'uniform float u_pointAttenuationMaxSize; \n' +
```

Comment:
That might be a little bit hard to read -- `u_pointSizeAndTilesetTimeAndAttenuation` maybe?